tests/test_airdrops.py

```diff
@@ -32,15 +32,26 @@ def test_csvs():
 
     for airdrop in airdrop_index['airdrops'].values():
         with open(airdrop['csv_path'], 'br') as f:
-            assert f.readline().startswith(b'address,amount'), f'{airdrop["csv_path"]} airdrop missing `address,amount` header'
-            for row_b in f:
-                row = row_b.decode('ascii')
-                # test that all addresses are checksummed
-                address = row.split(',')[0]
-                assert is_checksum_address(address), f'{airdrop["name"]} airdrop address {address} is not checksummed'
-
-                if row.endswith('\n'): # check that the CSV have LF EoL sequence
-                    assert row[-1] == '\n' and row[-2] != '\r', f'{row_b} in {airdrop["csv_path"]} should have LF EoL sequence'
+            csv_b = f.read()
+
+        # verify the csv_hash is correct
+        assert airdrop['csv_hash'] == hashlib.sha256(csv_b).hexdigest()
+
+        csv_rows_b = csv_b.split(b'\n')
+        assert len(csv_rows_b) > 0, f'{airdrop["csv_path"]} should have LF EoL sequence with multiple rows'
+
+        assert csv_rows_b.pop(0).startswith(b'address,amount'), f'{airdrop["csv_path"]} airdrop missing `address,amount` header'
+        for i, row_b in enumerate(csv_rows_b):
+            if row_b == b'' and i == len(csv_rows_b) - 1:
+                continue  # ignore empty row at the end
```

Comment:
why is there an empty row at the end? It would make sense to not have one. It's our data, so we control lenght and format.